gcs/src/main/java/com/google/cloud/hadoop/fs/gcs/GoogleHadoopFSInputStream.java

```diff
@@ -171,7 +185,7 @@ public synchronized int read(long position, byte[] buf, int offset, int length)
     Stopwatch stopwatch = Stopwatch.createStarted();
 
     int result = super.read(position, buf, offset, length);
-    readAPITrace(POSITIONAL_READ_METHOD, stopwatch, position, offset, length, result);
+    readAPITrace(POSITIONAL_READ_METHOD, stopwatch, position, offset, length, result, Level.FINE);
```

Comment:
out of curiosity why one read function is using level as `FINE` and other as `INFO`